### DISCUSSION TITLE
Flyte Sandbox: Minio password/key

### DISCUSSION THREAD

**Discussion by mpreusse** [flyteorg/flyte#4061] [Q&A] [answered]
What is the password/key/login for Minio in the Flyte Sandbox?

When I go to `http://localhost:30080/minio/login` after starting the sandbox I only see a login page.

**Comment by SDBlackwood** [comment, marked as answer]
Username: minio
Password: miniostorage

From https://docs.flyte.org/en/latest/community/contribute.html
